src/compute_centroid_stress_atom.cpp

```diff
@@ -226,22 +255,20 @@ void ComputeCentroidStressAtom::compute_peratom()
         stress[i][j] += cvatom[i][j];
   }
 
-  if (kspaceflag && force->kspace && force->kspace->compute_flag) {
-    double **vatom = force->kspace->vatom;
-    for (i = 0; i < nkspace; i++) {
-      for (j = 0; j < 6; j++)
-        stress[i][j] += vatom[i][j];
-      for (j = 6; j < 9; j++)
-        stress[i][j] += vatom[i][j-3];
-    }
-  }
+  //if (kspaceflag && force->kspace && force->kspace->compute_flag) {
+  //  double **vatom = force->kspace->vatom;
+  //  for (i = 0; i < nkspace; i++)
+  //    for (j = 0; j < 9; j++)
+  //      stress[i][j] += cvatom[i][j];
+  // }
```

Comment:
There will probably never be a centroid stress definition for kspace, and even if there was it would probably by a symmetric tensor, so the (currently commented out) code is a bit  misleading. Also, from a pure convenience point of view, if someone wanted to force the use of the original kspace stress definition for centroid/stress/atom (which is OK for total system heat flux), he would need to not only change the centroidstressflag, but also modify this part. My personal preference would be to revert the code or at least leave the original code that uses vatom commented out.